Audit alert v3 indicators enrichment fix (#32219)
demisto/content: Packs/PrismaCloudCompute/Playbooks/Prisma_Cloud_Compute_-_Audit_Alert_Vulnerabilities_Enrichment_README.md

| --- | --- | --- | --- |
| ImageVulnerabilities | Image vulnerabilities that were retrieved. |  | Optional |
| HostVulnerabilities | Host vulnerabilities that were retrieved. |  | Optional |

> | CVEsIndicatorsSeverities | A list of the image or host CVEs severities which the playbook creates indicators from.<br/>Available options \(not case sensitive\):<br/>- Critical<br/>- High<br/>- Medium<br/>- Low<br/>- Negligible<br/><br/>Default value: high, critical. | High, Critical | Optional |

Reviewer: ```suggestion
| CVEsIndicatorsSeverities | A list of the image or host CVE severities that the playbook creates indicators from.<br/>Available options \(not case sensitive\):<br/>- Critical<br/>- High<br/>- Medium<br/>- Low<br/>- Negligible<br/> | High, Critical | Optional |
```
